### PR TITLE
Fix: core/dhcp_server|adv_core/adv_dhcp_server up readme

### PR DESCRIPTION
**Pull request by oxedions**
Refer to #422

### CODE REVIEW

**Comment by gavillom** [comment]
Sounds good to me

File: roles/advanced-core/advanced_dhcp_server/readme.rst [hunk 1]
```diff
@@ -10,12 +10,20 @@ Features like shared_network, opt82, opt61 or snp/snponly roms are provided here
...
 
-Please read first documentation of the standard dhcp server role. Advanced dhcp server provides sames features than standard dhcp server role, but with more options.
+Please read first documentation of the standard dhcp server role. Advanced dhcp server provides same features than standard dhcp server role, but with more options.
 
 Dhcp will only take into account networks from the current iceberg, and with naming related to administration network (by default iceX-Y).
 
 Also, ensure dhcp is set to true for your network.
 
+Finally, note that the following parameters can be set in the inventory, to
+override defaults ones:
```
Comment (btravouillon) [inline roles/advanced-core/advanced_dhcp_server/readme.rst:20]: default ones

Comment (oxedions) [reply roles/advanced-core/advanced_dhcp_server/readme.rst]: Good catch. Corrected.

File: roles/advanced-core/advanced_dhcp_server/readme.rst [hunk 2]
```diff
@@ -10,14 +10,14 @@ Features like shared_network, opt82, opt61 or snp/snponly roms are provided here
 Instructions
 ^^^^^^^^^^^^
 
-Please read first documentation of the standard dhcp server role. Advanced dhcp server provides same features than standard dhcp server role, but with more options.
+Please read first documentation of the standard dhcp server role. Advanced dhcp server provides sames features than standard dhcp server role, but with more options.
```
Comment (btravouillon) [inline roles/advanced-core/advanced_dhcp_server/readme.rst]: same features was correct

Comment (oxedions) [reply roles/advanced-core/advanced_dhcp_server/readme.rst]: yeah, dont know Why I changed that. Correcting.

**Review by btravouillon — APPROVED** [review]
Looks good to me. 👍

**Pull request merged**